Gems/ScriptCanvas/Code/Include/ScriptCanvas/Grammar/AbstractCodeModel.cpp

```diff
@@ -4387,10 +4423,104 @@ namespace ScriptCanvas
             }
             else
             {
-                if (auto sourceVariable = ParseConnectedInputData(input, execution, nodes, FirstNode::Parent))
+                if (auto sourceVariableOperationData = ParseConnectedInputData(input, execution, nodes, FirstNode::Parent))
                 {
-                    execution->AddInput({ &input, sourceVariable, DebugDataSource::FromOtherSlot(input.GetId(), input.GetDataType(), sourceVariable->m_sourceSlotId) });
-                    CheckConversion(execution->ModConversions(), sourceVariable, execution->GetInputCount() - 1, input.GetDataType());
+                    if (!sourceVariableOperationData->m_smallOperations.empty())
+                    {
+                        Node* node = aznew Node();
+
+                        ExecutionSlotConfiguration inexconf;
```

Comment:
What's the motivation for the new Node here?

Reply:
It seems to be unused and leaked...

Reply:
This node gets added to the execution path to actually preform the operation.